setup.py

```diff
@@ -31,7 +31,8 @@
 """
 
 extras_require = {
-    "code_syntax_highlighting": ["pygments>=2.7.3"]
+    "code_syntax_highlighting": ["pygments>=2.7.3"],
+    "wavedrom": ["wavedrom; python_version>='3.7'"]
```

Comment:
Have a link to docs on `python_version` within extras? Haven't seen it before.

Reply:
I don't have anything concrete that says this is allowed/disallowed.

In the [setuptools docs](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies) on the `extras_requires` key, it shows the following example configuration:
```python
setup(
    name="Package-A",
    ...,
    extras_require={
        "PDF": ["ReportLab>=1.2", "RXP"],
    },
)
```
And I took the `ReportLab>=1.2` to mean that it supported the full package specifying syntax that `install_requires` does.
In my testing, this does seem to be the case.

This [stackoverflow answer](https://stackoverflow.com/a/65936306/) claims that this is completely possible and links to the [imgui](https://github.com/pyimgui/pyimgui/blob/74c359702542c4319f4541bab6825597991e980a/setup.py#L111) library as an example

Reply:
Makes sense, thank you for this